### DISCUSSION TITLE
FaceSwap not working

### DISCUSSION THREAD

**Discussion by sambhavv22** [lllyasviel/Fooocus#1087] [Q&A]
Whenever I try to do a FaceSwap the program processes the prompt but as soon as it loads it to the GPU it just crashes.
Specs: Ryzen 5 4600H, GTX 1650Ti 4 GB VRAM, 16 GB 3200Mhz Ram

![Screenshot (54)](https://github.com/lllyasviel/Fooocus/assets/105424680/f34d5f11-c19f-4531-9a19-8d8dace5b74d)

**Comment by hotlml** [comment]
> GTX 1650Ti 4 GB VRAM
The graphics card is not performing well. Please replace it with an RTX 3060.
